.github/workflows/aws-deploy.yml

```diff
@@ -0,0 +1,59 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/master/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+
+name: Deploy to AWS
+
+on:
+  workflow_run:
+    workflows: [ "validate" ]
+    branches: [main]
+    types:
+      - completed
+
+concurrency:
+  group: ${{ github.workflow }}
+
+jobs:
+  cdk-deploy:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Static Analysis
+        uses: pre-commit/action@v3.0.0
+
+      - name: Install dependencies
+        run: pip install -r requirements.txt -r requirements-dev.txt
+
+      - name: Run unit tests
+        run: python -m pytest tests/ -s -v
+
+        # Authenticate to AWS using GitHub OIDC
+      - name: Assume AWS Role
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-region: us-east-1
+          role-to-assume: arn:aws:iam::<account-number>:role/<role-name>
+          role-session-name: GitHubActions-${{ github.repository_owner }}-${{ github.event.repository.name }}-${{ github.run_id }}
+          role-duration-seconds: 1200
+
+        # https://github.com/marketplace/actions/aws-cdk-github-actions
+      - name: cdk synth
+        uses: youyo/aws-cdk-github-actions@v2
+        with:
+          cdk_subcommand: 'synth'
+          cdk_args: '--context env=dev --debug'
+
```

Comment:
Not sure why uncommenting code is part of the deployment process. Is it to allow testing more of the pipeline (say, in a PR against the main branch) before deploying to an AWS account (say, in another branch)?  If so, can you please add a comment to this effect in the README?

Reply:
The purpose of a template repo is to just get users started.  We are not deploying this project to a real AWS account.  that's why it's commented out.  Users can uncomment and update the `role-to-assume` in this file with a real/valid role to allow deployment to a real AWS account.

Reply:
Oh, right.  My mistake. (I thought I was looking at the `data_curator-infra` repo'.